terms.html

```diff
@@ -53,6 +61,9 @@
   <dd>
 A thing with distinct and independent existence, such as a person,
 organization, or device that performs one or more roles in the ecosystem.
+For accountability and other purposes, it can be relevant to know 
+whether an entity (for instance, a person, or a device) performs such roles by itself,
+or on behalf of some other entity (for instance, a person, or an organization).
```

Comment:
These are details that don't add much value to the definition. While they are true, adding them to the definition weakens the ability to understand and convey the definition easily.